packages/react/src/autocomplete-content/AutocompleteContent.css.ts

```diff
@@ -0,0 +1,39 @@
+import { recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    style({
+      maxHeight: "var(--radix-popover-content-available-height)",
+      minWidth: "var(--radix-popover-trigger-width)",
+      position: "relative",
```

Comment:
doesn't popover already have this - do we need to duplicate this here?

Reply:
We aren't reusing our popover rather the radix popover as we need to force-mount.